ecommerce/coupons/views.py

```
@@ -29,9 +31,9 @@
 Voucher = get_model('voucher', 'Voucher')
 
 
-def get_voucher(code):
+def get_voucher_from_code(code):
```

Please create a ticket to refactor this function. We should raise an exception in exceptional cases (e.g. the voucher does no exist, or the voucher has no associated products). Using special return values (such as `None`) is prone to errors, especially when those special values are not documented (which they aren't in this case).